README.md

```diff
@@ -0,0 +1,128 @@
+# Mountsion CBS Secondary Website
+
+This project is a website that is being made by 4 students from Mountsion CBS Secondary along with the help of RedHat Waterford
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. Here's how you can get set up
```

Comment:
This section can be replaced by linking to the tutorial we have in the [docs](https://github.com/MountSionCBSSecondary/SchoolWebsite/blob/master/docs/Tutorial.md)